src/csync/csync_private.h

```diff
@@ -103,14 +104,15 @@ struct csync_s {
     int lastReturnValue;
   } statedb;
 
+  typedef std::map<QByteArray, std::unique_ptr<csync_file_stat_t>> FileMap;
```

Comment:
could also be std::unordered_map

Reply:
I had a hard time making that call since we iterate those trees, but I realize now that the previous tree was ordered by phash so I guess the order didn't really matter.

Would there be any advantage to use `std::map`?

Reply:
Meh I just tried and it complains about `error: implicit instantiation of undefined template 'std::__1::hash<QByteArray>'`.
I'd have to bridge qHash on top of std hash or something like that I guess, not sure if this is worth it.